packages/gatsby-worker/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "gatsby-worker",
+  "description": "Utility to create worker pools",
+  "version": "0.0.0-next.0",
```

Comment:
Early on I would prefer to keep it `<1` as we might quickly introduce breaking changes as we start using worker pool and discover new needs or APIs (that's what <1 is for). 1 is when API becomes actually stable enough

Reply:
Curious how this will play with lerna minor bump - I think it should do `0.1.0-next.0` for the next branch cut? If so, this sounds good to me.

Reply:
https://github.com/gatsbyjs/gatsby/commits/master/packages/gatsby-plugin-graphql-config/package.json is example of `0.X.Y` package we have and it does bump minor for each release. We want `gatsby-worker` to be actual dep only of `gatsby` core package - if we will need it in `gatsby-cli` for structured logs, we will import it from gatsby/internal there. Messaging (but this will only need messaging and not queueing up tasks)